src/hooks/useBreakpoints.ts

```diff
@@ -10,6 +18,8 @@ const BREAKPOINT_WIDTH: Record<Breakpoints, number> = {
   xl: 1280,
   "2xl": 1536,
   "3xl": 1920,
+  largedisplay: 2080,
+  fourK: 3000,
```

Comment:
use standard values and consistent namings

```suggestion
  "4xl": 2560,
  "4k": 3840,
```